discord_bots/views/configure_category.py

```diff
@@ -0,0 +1,319 @@
+import logging
+
+from discord import (
+    ButtonStyle,
+    Client,
+    Colour,
+    Embed,
+    Interaction,
+    SelectOption,
+    TextStyle,
+)
+from discord.ui import button, Button, Modal, Select, TextInput
+
+from discord_bots.models import Category
+from discord_bots.views.base import BaseView
+from discord_bots.views.confirmation import ConfirmationView
+
+_log = logging.getLogger(__name__)
+
+
+class CategoryConfigureView(BaseView):
+    def __init__(self, interaction: Interaction, category: Category):
+        super().__init__(timeout=300)
+        self.value: bool = False
+        self.category: Category = category
+        self.interaction: Interaction = interaction
+        self.embed: Embed
+        self.add_item(CategoryRatedSelect(self))
+
+    @button(label="Set Name", style=ButtonStyle.primary, row=0)
```

Comment:
I think I prefer the "grey" button style for all of these options, and Blue/grey for save/cancel